modes/config-mode/src/index.js

```diff
@@ -60,21 +60,17 @@ const modesFactory = async (modes, extensions) => {
   // TODO - iterate over the set of actual modes, and import only the required ones once.
   // Creates a mode called configPoint-base-@ohif/mode-longitudinal
   createDerivativeMode(await import("@ohif/mode-longitudinal"));
+  createDerivativeMode(await import("@ohif/mode-tmtv"));
```

Comment:
This allows extending the tmtv mode as well

Reply:
as said before we should have a way to avoid coupling to ohif